endurance-page-cache.php

```diff
@@ -1124,7 +1124,11 @@ public function toggle_nginx( $new_value = 0 ) {
 		 */
 		public function update( $checked_data ) {
 
-			$muplugins_details = wp_remote_get( 'https://api.mojomarketplace.com/mojo-plugin-assets/json/mu-plugins.json' );
+			$muplugins_details = get_transient( 'mojo_plugin_assets' );
+			
+			if ( ! $muplugins_details ) {
+				$muplugins_details = wp_remote_get( 'https://api.mojomarketplace.com/mojo-plugin-assets/json/mu-plugins.json' );
+			}
 
 			if ( is_wp_error( $muplugins_details ) || ! isset( $muplugins_details['body'] ) ) {
 				return $checked_data;
```

Comment:
Because the transient is saved as the json_decode of the body it is returned early here.